test/renderer_test.exs

```diff
@@ -82,4 +82,10 @@ defmodule RendererTest do
 
     assert render(slim) == ~s(<p class="test"></p>)
   end
+
+  test "render tag with boolean attribute" do
+    assert render(~s(div [ab="ab" a] a)) == ~s(<div ab="ab" a>a</div>)
```

Comment:
As far as I know there is no difference between unquoted attribute and quoted attribute with the same value. Also there is no support for unquoted attributes in slim. If somebody for some reason wants unquoted attribute in result html, he can use inline html for that.